CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `XRPTransactionType` now includes an `AccountSetValue`.
 - `XrpTransactionType` replaces the now deprecated `XRPTransactionType`.
 - `XrpTransaction` replaces the now deprecated `XRPTransaction`.
+- A new method, `cryptoAddressForPayId`, replaces the now deprecated `addressForPayId` method in `PayIdClient`.
```

Comment:
What is wrong with the name `addressForPayId`? What's the benefit of adding `crypto` to the name of the method?

Reply:
Does it maybe assume a `CryptoAddressDetails` object? Or is this just so we can "deprecate" `addressForPayId`, without making a breaking change to an existing function?

Would we theoretically have `achAddressForPayId`, `sepaAddressForPayId`, `creditCardDetailsForPayId`?

Reply:
It does assume crypto details. Also it takes a parameter. 

This PR is fundamentally changing this method.